src/components/Feedback/Feedback.js

```diff
@@ -1,162 +1,15 @@
-          topic: previous.requestForm.topic,
-          title: previous.requestForm.title,
-          contactEmail: part,
-          requestAdditions: previous.requestForm.requestAdditions
-        }
-      }));
-      break;
-    default:
-      break;
-    }
-  }
-  render() {
-    return (
-      <footer style={fooStyles.foo} className="text-center col-xs-12">
-        <form
-          name="requestUs"
-          data-netlify="true"
-          netlify-honeypot="bot-field"
-          method="POST"
-          action="https://formspree.io/your@email.com"
-          netlify
-          onSubmit={this.handleSubmit}
-          className="col-md-4 col-md-6 make-request"
-        >
-          <div>
-            <h4>Форма зворотнього зв&apos;язку</h4>
-          </div>
-          <div className="form-group">
-            <input
-              type="text"
-              onChange={e => { this.handleChange(e, 'topic'); }}
-              value={this.state.requestForm.topic}
-              placeholder="Тема"
-              className="form-control inpt-round"
-              name="topic"
-            />
-          </div>
-          <div className="form-group">
-            <input
-              type="text"
-              onChange={e => { this.handleChange(e, 'title'); }}
-              value={this.state.requestForm.title}
-              placeholder="Питання"
-              className="form-control inpt-round"
-              name="title"
-            />
-          </div>
-          <div className="form-group">
-            <input
-              type="text"
-              onChange={e => { this.handleChange(e, 'requestAdditions'); }}
-              value={this.state.requestForm.requestAdditions}
-              placeholder="Додаткові дані"
-              className="form-control inpt-round"
-              name="requestAdditions"
-            />
-          </div>
-          <div className="form-group">
-            <input
-              type="email"
-              onChange={e => { this.handleChange(e, 'contactEmail'); }}
-              value={this.state.requestForm.contactEmail}
-              placeholder="Контактна адреса"
-              className="form-control inpt-round"
-              name="contactEmail"
-            />
-          </div>
-          <button
-            className="btn-lg btn-default"
-            style={fooStyles.btn}
-            value="submit"
-          >Надіслати
-          </button>
-        </form>
-        <MessageSnippet message={this.state.messageSuccess} />
-      </footer>
-    );
-  }
-}
+export default props => (
+  <form
+    action={`https://formspree.io/${props.email}`}
+    method="POST"
+  >
+    <label htmlFor="form-message" >Повідомлення:</label>
+    <textarea id="form-message" name="message" required placeholder="Повідомлення" />
+    <label htmlFor="form-contact">Email, щоб ти точно отримав відповідь</label>
+    <input id="form-contact" type="text" name="_replyto" placeholder="example@test.com" required />
```

Comment:
It's not necessary to set the type="text" directly... you can omit it

Reply:
True